docs/products/grafana/reference/plugins.rst

```diff
@@ -1,7 +1,7 @@
 List of available plugins
 =========================
 
-The following plugins are available in all installations of Aiven for Grafana, and they are updated at intervals. To request additional plugins, please reach out to our support team.
+The following plugins are available in all installations of Aiven for Grafana, and they are updated at intervals. To request additional plugins, please reach out to our support team: support@aiven.io.
```

Comment:
in the context of an e-mail one must use something like this: 
```
You can contact the author via e-mail at <alex.dupuy@mac.com>.
```

for more info: https://gist.github.com/dupuy/1855764#hyperlink-urls

Reply:
This comment is wrong. I have visited the CONTRIBUTING page and read about e-mail and the way it should be added is as ``someone@example.com`` for example.